### ISSUE TITLE
Java - Accuracy issues for Estimate Cost 

### ISSUE THREAD

**Issue by unclebinary1001** [StateFarmInsCodingCompetition/2023-StateFarm-CodingCompetition#9]
the `getTotalClaimCostForDisaster` method is of float data type so accuracy does not match expected value in TestSet2 so `test4_getTotalClaimCostForDisaster` fails for `getTotalClaimCostForDisaster(56).` 

![image](https://github.com/StateFarmInsCodingCompetition/2023-StateFarm-CodingCompetition/assets/69122375/669f45d2-6d45-4e4e-801f-96f5b21c2c6a)

I tried changing `estimate_cost ` data type in Claim.java and the return types of the methods that depend on it from `Float` to `Double`. This fixes the test and it works expected

**Comment by J1411** [comment]
Thanks for bringing this up. We're double checking and looking into it. Please continue on and we'll get back to you shortly

**Comment by J1411** [comment]
@unclebinary1001 You are right, looks like a `float`/`Float` doesn't have enough precision and we get the wrong expected value back. Thanks for catching this. I'll had it to our change log and push up a fix. Though it is late in the competition, hopefully this will help others working in Java too.

**Comment by unclebinary1001** [comment]
Sounds good.

**Comment by proesslet** [comment]
I believe something similar may be happening in Test Set 3 for Test 11. 
![Screenshot 2023-10-14 at 11 10 15 PM](https://github.com/StateFarmInsCodingCompetition/2023-StateFarm-CodingCompetition/assets/10239258/9ab51a6e-2e0d-412a-a7bc-d6bb827915cf)
